core/kazoo_number_manager/src/carriers/knm_telnyx.erl

```diff
@@ -15,6 +15,7 @@
 -export([is_number_billable/1]).
 -export([should_lookup_cnam/0]).
 -export([check_numbers/1]).
+-export([check_portability/1]).
```

Comment:
Can you update `knm_gen_carrier` to add `check_portability/1` as an optional callback?